content/Foundation/Trademark/home.pml

```diff
@@ -12,8 +12,10 @@
   in your work, following the usage rules below.
 </p>
 <p>
-  The Linux Foundation Projects (LF Projects, LLC) has applied to register the
-  seL4 trademark in the United States and other countries.
+   The Linux Foundation has registered seL4<sup>&reg;</sup> as a
+  trademark in the United States. Please use &reg;
+  with the first mention of seL4 in all materials.
+  The Linux Foundation Projects (LF Projects, LLC) has also applied to register the seL4 trademark in other countries.
```

Comment:
Probably don't need that sentence any more if we add "other countries" to the previous one.

Reply:
Ok. Gernot said there was some other required wording in the history of the pml, but I didn't see anything else that we might need to add.

Reply:
Ok, let me check

Reply:
I don't see anything else in the history either.